src/PeopleLookup.Mvc/Models/AuthSettings.cs

```diff
@@ -7,8 +7,11 @@ namespace PeopleLookup.Mvc.Models
 {
     public class AuthSettings
     {
+        public string Authority { get; set; }
         public string IamKey { get; set; }
         public string CasBaseUrl { get; set; }

```

Comment:
Remove?

Reply:
authority is important and needed

Reply:
@srkirkland Sorry, clicked on the wrong line. Meant the CasBaseUrl  here.